src/Tasks/Common/Resources/xlf/Strings.de.xlf

```diff
@@ -356,6 +356,21 @@ The following are names of parameters or literal values and should not be transl
         <target state="translated">NETSDK1059: Das Tool "{0}" ist jetzt im .NET Core SDK enthalten. Informationen zum Auflösen dieser Warnung sind unter https://aka.ms/dotnetclitools-in-box verfügbar.</target>
         <note>{StrBegin="NETSDK1059: "}</note>
       </trans-unit>
+      <trans-unit id="ReadyToRunCompilerOutput">
+        <source>NETSDK1088: {0}</source>
+        <target state="new">NETSDK1088: {0}</target>
+        <note>{StrBegin="NETSDK1088: "}</note>
+      </trans-unit>
+      <trans-unit id="ReadyToRunNoValidRuntimePackageError">
+        <source>NETSDK1086: Unable to optimize assemblies for performance: a valid runtime package was not found. Either set the ReadyToRun property to false, or use a supported runtime identifier when publishing.</source>
+        <target state="new">NETSDK1086: Unable to optimize assemblies for performance: a valid runtime package was not found. Either set the ReadyToRun property to false, or use a supported runtime identifier when publishing.</target>
```

Comment:
Not really sure what's the process of getting these messages translated. There are other english strings too in these non-english xlf files